### PR TITLE
Pull translations on iOS CI

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for b59f50fabcc4e56cc2386584d54c824306c00787:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-643928eb44125d09d002df3d912556c03645b921-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-643928eb44125d09d002df3d912556c03645b921-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-643928eb44125d09d002df3d912556c03645b921-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-643928eb44125d09d002df3d912556c03645b921-x86.apk)

**Comment by m-kuhn** [comment]
Fixes https://github.com/opengisch/QField/issues/2149 ?

File: .github/workflows/ios.yml [hunk 1]
```diff
@@ -24,17 +24,24 @@ jobs:
         qt-version: ['5.14.2']
         build-type: ['Release']
         bitcode: ['FALSE']
+    env:
+      TX_TOKEN: ${{ secrets.TX_TOKEN }}
```
Comment (m-kuhn) [inline .github/workflows/ios.yml:28]: Can this be set just for the specific step instead of global?

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f3155ea73c248f759143ffc7a8711ae1b07ba1ed:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-a36b7202637ad6e2a930b944eb61e5ae71e04542-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-a36b7202637ad6e2a930b944eb61e5ae71e04542-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-a36b7202637ad6e2a930b944eb61e5ae71e04542-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2150-a36b7202637ad6e2a930b944eb61e5ae71e04542-x86.apk)

**Pull request merged**